controller/requirements.txt

```diff
@@ -21,3 +21,4 @@ setproctitle==1.1.8
 static==1.1.1
 South==1.0.2
 python-ldap==2.4.19
+marathon
```

Comment:
@smothiki We should probably lock this to a specific version of `marathon` so they don't break our code with updates.